pom.xml

```diff
@@ -48,7 +48,7 @@
             <dependency>
                 <groupId>com.typesafe.play</groupId>
                 <artifactId>play_2.12</artifactId>
-                <version>2.6.25</version>
+                <version>2.8.13</version>
```

Comment:
shouldn't it be 2.8.15 ?

Reply:
it is the prev change. in latest one it is already 2.8.15